### ISSUE TITLE
Tell the story of the Participation Team

### ISSUE THREAD

**Issue by jgmac1106** [mozilla/participation-org#203]
**Edited: Goal was changed to reflect monthly blog challenge** 
# Get the Participation Team Blogging
# Success
- Everyone posts a monthly blog
-  Everyone is added to the Participation Planet
# Roles
- owners @geroter and @emmairwin 
- involved @jgmac1106  Daniele Scasciafratte 
- informed @LucyeoH 
# Required
- Monthly topic question
- Hanging out reading everyone's writing on the planet.

**Comment by deimidis** [comment]
i believe is not Giovanny. Giovanny is the Colombian guy. Danielle is the Italian one

**Comment by jgmac1106** [comment]
I thought it was Danielle as well just didn't see the name on an email. I will fix it. Thanks @emmairwin and @geroter let me know if I did this right. Hope I didn't step on toes just didn't want to forget.

**Comment by ioana-chiorean** [comment]
The check boxes - [ ] after the \*  - syntax \* [ ] 
- [x]

**Comment by geroter** [comment]
This is awesome!!!

"Template" below...

## #MyMozFest Blog post

### At Mozfest I…

[Talk about a conversation with someone, an experience you had, etc]

#### Teachable Moment

[The experience that made you a better teacher, facilitator or mentor, and why you think that is]

### From that conversation it made me think about…

[what came up in your mind as you had this conversation]

### Which is relevant to Mozilla because…

[how this could be relevant or useful in your community]

### My call to action is...

[here's how you might help, here's what I'd like your thoughts on]

**Comment by Jobava** [comment]
OK, I sort of stumbled into a way to host blogs on Github.

Example: https://jobava.github.io/

Example post: https://jobava.github.io/mozilla/2015/11/09/hello/

It needs a Disqus account to support comments and a FeedBurner account for a feed of the posts.

The Jekyll framework allows you to edit blog posts using only Markdown.

I could create a tutorial once I figure things out myself, if you'd like.

Jobava

**Comment by jgmac1106** [comment]
@Jobava  I know a lot of people who you github and Jekyll to host blogs. I myself am thinking of learning and migrating there.

I think it would be too heavy of a lift for many of the participation leaders.

Here is a form I drafted that I would include in an email to the other participation leaders. Please take a look.

https://docs.google.com/forms/d/1jefLk4V5QD3ZSe-Cce0jHMgAJvSotW9__YYdS05C7Kk/viewform?usp=send_form

Basically I am offering to either host a registered domain on my server or give them a subdomain of participation.community.

**Comment by Jobava** [comment]
Hey @jgmac1106 

Does participation.community have a wildcard SSL certificate? Otherwise, we're stuck with plain http, or wait for Let's Encrypt (which requires a separate cert per subdomain nonetheless).

I'd suggest a url like: participation.mozilla.org/username which then needs a cert for just that single domain/subdomain.

**Comment by jgmac1106** [comment]
That sounds like a plan I did not want to suggest using a Mozilla address
since I am NOT staff so I just registered a domain but I could never afford
the wildcard SSL I keep one for my other servers but I would not be able to
get one for this domain if we can set one up or if somebody has a better
idea I am all for it I will change the form to just stay at your URL and we
can get them on a planet but not offer any kind of hosting I am absolutely
willing to post them on my servers or my cloud base I use reclaim hosting
but maybe just encourage them to use a free spot miles take exit 69 for
Connecticut 9 north toward Essex Hartford Paramore GPS came through my
voice typing sorry about the typos will fix went home

**Comment by jgmac1106** [comment]
@Jobava 

Thank you for the update. I had no idea about that. I make all my class domains class.mynetwork.org.  I do have a wildcard ssl certificate but that is on another domain. I can afford to donate some extra hosting but not another wildcard.

Security is so expensive.

Here the issue then.
- Many of the participation leaders can't afford hosting or even domain registration.
- I want to create a pathway of portability that eventually moves to their own domain.

Maybe it is best to have them just use a commercial service. Learning to host a jekyll site on gitpages is passed many skill levels (including my own).

I use Reclaim Hosting. I was just going to create a ton of subdomains, but I had no idea that wasn't secure. I do it all the time.

Why I volunteer. I learn so much.

**Comment by jgmac1106** [comment]
...or if you want to teach me the GitHub pages way using Jekyll we could work together on the tutorials. Learning out loud makes the best lessons. 

Realized that was your original idea. Makes sense.

**Comment by jgmac1106** [comment]
Email sent

**Comment by emmairwin** [comment]
I added 'teachable moment' to the template. I would really love people to think about what they learned, how they learned it and how we can share what we learn with others moving forward.  This will be in my blog post as well, which will post soon.

**Comment by jgmac1106** [comment]
@emmairwin noted that the best way to add to the Participation Planet will be through Bugzilla.

**Comment by emmairwin** [comment]
Direct link to add your blog:   https://bugzilla.mozilla.org/enter_bug.cgi?product=Websites&component=planet.mozilla.org

Add the feed URL of your blog for participation  (you need a tag called participation), and specific this is for the participation planet.  I'll promote this a bit in a group email as well.  Thanks @jgmac1106 !

**Comment by Mte90** [comment]
My report in english: http://www.mte90.net/en/2015/11/mozilla-festival-2015-i-was-there/
My report in italian: http://www.mte90.net/2015/11/mozilla-festival-2015-io-cero/

For the participation planet we need to create a a ticket for everyone?

For the help on create a blog I'm avalaible and i want to promote https://mte90.github.io/Participation-Lead-Map/

**Comment by ioana-chiorean** [comment]
> For the participation planet we need to create a a ticket for everyone?

yes

**Comment by jgmac1106** [comment]
Hello all,

About a week left on our first participation challenge. We have about 25%
of the first #mymozfest stories.

We have heard a few people are hesitant. That is okay. Writing is tough.

_Feeling vunerable/shy_
It was hard for me to get started blogging as well. I was afraid to put
stuff out there. Luckily I realized I am pretty much just writing for
friends. I have been blogging a long time and few people see my work. My
writing is shared among my network. It helps out  my friends.

_What you can do?_
Do a photographic essay. Who said you had to write a ton of words. Share
all your photos from MozFest and then answer George's prompts in one
sentence.
Be short. Shorter writing pieces do better,
Share with us before publishing. Want us to see what you wrote. Email it to
us or share on Telegram.
Just document what you do. Afraid you have nothing to say? Just describe
what you did at mozfest. Documenting our learning is the ultimate goal.

_Not good at English_
Who cares. We are the ignorant one's who only speak one language. Do not
cater to us.

You are participation leader. Your goal is to increase involvement across
all Moz channels. This means we NEED you to write in you localized
language. The next leader for your community isn't from an English only
country. They are from your hom town, an internet cafe in a nearby city, or
even in a remote village.

An English only Web is not an Open Web. Part of Mozilla's goal is to
protect the diversity of the web and resist the hegemonic forces that have
created a monolinguistic space.

When you blog in your native tongue you are celebrating your culture and
helping to unleash the next wave of open.

**Comment by emmairwin** [comment]
@jgmac1106     This is great advice  - I wonder if we should think about how we can capture this type of advice for the whole cohort in future, almost like a Participation Leadership 101 .   

also +10000  on all  these points: write and share in the ways that make you feel creative, free and heard.

**Comment by anupkumarmishra** [comment]
My Blogpost http://dare2inanup.blogspot.in/2015/11/mozfest-2015-moz-experience-being.html @geroter @emmairwin

**Comment by emmairwin** [comment]
aswesome @anupkumarmishra !

**Comment by yofiesetiawan** [comment]
My blogpost, we have it in 2 languages.
http://mozilla.or.id/en/blog/news/104-mozilla-festival-come-with-an-idea-leave-with-a-community.html
http://mozilla.or.id/id/blog/berita/104-mozilla-festival-come-with-an-idea-leave-with-a-community.html

@geroter @emmairwin

**Comment by emmairwin** [comment]
@yofiesetiawan - completely in awe.  , so humbled as well.  some great photos!

**Comment by umeshagarwal** [comment]
Hi ,
Here is my blog: https://fosswithumesh.wordpress.com/2015/11/28/mozilla-festival-2015-lets-start-again/

**Comment by helitomato** [comment]
My blog post - http://bc.mozillacanada.org/mozfest-2015-design-thinking-maximize-resources-in-your-community/

**Comment by birajkarmakar** [comment]
Here is my blog https://birajkarmakar.wordpress.com/2015/12/03/mozfest-2015-experience/

**Comment by fiji-flo** [comment]
Finally my post: http://fiji-flo.github.io/

**Comment by sara-mansouri** [comment]
My blog posts :)
https://medium.com/@SaraMansouri/participation-leadership-in-mozfest-2015-34d05ec71b33#.wmc2n8zcd

https://medium.com/@SaraMansouri/holding-a-flexagon-session-in-mozfest-2015-1dbc845cb288#.rkcfr76zl

**Comment by sudosumanth** [comment]
My blog post - https://sumanthmoz.wordpress.com/2015/11/09/mozfest-2015/

**Comment by shadowcrow** [comment]
https://medium.com/@shadowcrow_/mymozfest-2015-d3c70c81049b#.8d0qowxpw my post in Chinese.
Wish I can finish English version after new year

**Comment by tanhaislam** [comment]
my blogpost: 
In english:
https://medium.com/@pink.devil.tanha/mozilla-festival-2015-this-time-i-made-it-f49ee4cd5be8#.z0exbpoij
In Bengali: (on a leading newspaper)
http://www.prothom-alo.com/we-are/article/690784/%E0%A6%AE%E0%A7%87%E0%A6%BE%E0%A6%9C%E0%A6%BF%E0%A6%B2%E0%A6%BE%E0%A6%B0-%E0%A6%89%E0%A7%8E%E0%A6%B8%E0%A6%AC%E0%A7%87?print=1#jadewits_print

**Comment by ShaguftaMethwani** [comment]
A little late to the party, infact reaching after it got over i guess, but here it is- https://bit.ly/my-mozfest . Got delayed due to exams!

**Comment by jgmac1106** [comment]
I was going to close this issue but many people have asked to keep the blog challenge going. I am thinking a once the month prompt or idea.

**Comment by ShaguftaMethwani** [comment]
That would be interesting. Once a month sounds like a great idea to me

**Comment by LucyeoH-zz** [comment]
@jgmac1106 looks like this has lost some momentum. Can it close? Or would you like to extend it to Jan 29th or the end of the quarter (March 31st)?

**Comment by jgmac1106** [comment]
Yes you can close. We moved over to discourse for the monthly challenges.
Got a few in Dec on working open. Jan will ve a privacy prompt.
